dist/index.html

```diff
@@ -2,11 +2,197 @@
 <html>
```

Comment:
<!DOCTYPE html> I don't know if we do this if we're using webpack but I know that's something we use for webpages